src/ui/common/src/components/pages/workflows/index.tsx

```diff
@@ -49,34 +54,41 @@ const WorkflowsPage: React.FC<Props> = ({ user, Layout = DefaultLayout }) => {
     </Box>
   );
 
-  const workflowList =
-    allWorkflows.workflows.length > 0 ? (
-      <Box sx={{ maxWidth: '1000px', width: '90%' }}>
-        {allWorkflows.workflows.map((workflow, idx) => {
-          return (
-            <React.Fragment key={idx}>
-              <Box my={2}>
-                <WorkflowCard workflow={workflow} />
-              </Box>
-              {idx < allWorkflows.workflows.length - 1 && <Divider />}
-            </React.Fragment>
-          );
-        })}
+  const displayFilteredWorkflows = (workflow, _) => {
+    return (
+      <Box my={2}>
+        <WorkflowCard workflow={workflow} />
       </Box>
-    ) : (
-      <Typography variant="h5">
-        There are no workflows created yet. Create one right now with our{' '}
-        <Link href="https://github.com/aqueducthq/aqueduct/blob/main/sdk">
-          Python SDK
-        </Link>
-        !
-      </Typography>
     );
+  };
+
+  const noItemsMessage = (
+    <Typography variant="h5">
+      There are no workflows created yet. Create one right now with our{' '}
+      <Link href="https://github.com/aqueducthq/aqueduct/blob/main/sdk">
+        Python SDK
+      </Link>
+      !
```

Comment:
This formatting is kind of strange. I think the linter might have done this. Is there a way you can put this "!" with the "Python SDK" text?

Reply:
Do you mean include it in the link?
```
<Link href="https://github.com/aqueducthq/aqueduct/blob/main/sdk">
Python SDK!
</Link>
```

Reply:
Can do that, or also put inside of a <span>!</span> after the link tag.